nbs/core.ipynb

```diff
@@ -57,7 +57,7 @@
     "import os\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #223.            use_init_models : bool, option (default=True)
</pre><p>I think there is a minor typo here, <code>optional</code> instead of <code>option</code>. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/644/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>